src/js/components/datepicker/datepicker_state.js

```diff
@@ -4,31 +4,47 @@ import dayjs from 'dayjs';
 
 export default Backbone.Model.extend({
   initialize() {
-    this.set('selectedDate', this.dayjs('selectedDate'));
-    this.setCurrentMonth(this.dayjs('currentMonth'));
+    this.setCurrentMonth(this.get('currentMonth'));
```

Comment:
May not be the correct approach, but we do have a scenario where a `currentMonth` can be passed in without a `selectedDate` or `selectedMonth`. This would account for that, but be overridden if either selected option is passed.

Reply:
That works.  FWIW.. I believe the first attribute in initialize is the data so rather than the `get`s it could be `initialize({ currentMonth, selectedDate, selectedMonth })`